components/ui/DialogComponent.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Story, Meta } from "@storybook/react";
+import DialogComponent, { DialogComponentProps } from "./DialogComponent";
+
+export default {
+  title: "UI/Dialog",
+  component: DialogComponent,
+} as Meta;
+
+const Template: Story<DialogComponentProps> = (args) => (
+  <DialogComponent {...args}>
+    <div className="w-40 cursor-pointer">
+      <p className="text-center">This is the Content.</p>
+    </div>
+  </DialogComponent>
+);
+
+export const Primary = Template.bind({});
```

Comment:
You're looking like a storybook pro, when you merge your PRs in ask me to unlock a storybook badge for you.